### DISCUSSION TITLE
Время между выдачей и ...

### DISCUSSION THREAD

**Discussion by ItzIsonn** [iBuseWinner/Reputation#14] [Ideas]
Предлагаю в конфиг добавить функцию изменения времени между выдачей одного вида репутации (полож./отриц.). Ведь как я понимаю сейчас это значение стоит на 1 день, то есть выдал игроку Test +1 репутацию, завтра опять можешь ему же выдать +1.

И еще сделать в конфиге вкл/откл параметр: может ли игрок выдать другому игроку +1, а потом забрать (-1)

**Comment by iBuseWinner** [comment]
Сейчас нет никаких ограничений по времени. Запишу эту идею, возможно реализую когда-нибудь.

По поводу второго - уже записано в планы, пока нет времени.

**Comment by ItzIsonn** [comment]
Наверное вы меня неправильно поняли насчет первого.
Я не могу выдать одному игроку подряд +1, но если подождать день, то снова могу выдать +1
